_plugins/jekyll-notranslate.rb

```diff
@@ -0,0 +1,85 @@
+module Jekyll
+  module NoTranslate
+    # `text` is in HTML
+    def no_translate(text, words)
+      escaped_words = words.map { |word| Regexp.escape(word) }
+      text = text.gsub(
+        /(\b#{escaped_words.join("|")}\b)(?![^<]*?>)/i,
+        "<span class=\"notranslate\">\\1</span>"
+      )
+      text
+    end
+  end
+end
+
+if $0 == __FILE__
```

Comment:
running `ruby _plugins/jekyll-notranslate.rb` should run the test suite below. Make sure it's green before merging this patch.